components/director/internal/model/labeldef.go

```diff
@@ -5,6 +5,57 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	ScenariosKey = "scenarios"
+)
+
+var (
+	ScenariosSchema = map[string]interface{}{
+		"type":        "array",
+		"minItems":    1,
+		"uniqueItems": true,
+		"items": map[string]interface{}{
+			"type": "string",
+			"enum": []string{"DEFAULT"},
+		},
+	}
+	ScenariosSchemaDefinitionSchema = map[string]interface{}{
```

Comment:
maybe better name will be `SchemaForScenariosSchema`, please add a comment to that variable to explain its purpose